fhir-server/liberty-config/config/default/fhir-server-config.json

```diff
@@ -111,7 +111,6 @@
             },
```

Comment:
Can persistence/jdbc/bootstrapDb and persistence/jdbc/bootstrapDataSourceBase be removed since they are removed from FHIRConfiguration.java?

Reply:
done